frontend/src/features/admin-form/settings/SettingsPage.tsx

```diff
@@ -106,7 +110,7 @@ export const SettingsPage = (): JSX.Element => {
           <TabPanel>
             <SettingsWebhooksPage />
           </TabPanel>
-          {user?.betaFlags?.payment && (
+          {(user?.betaFlags?.payment || paymentGlobalBeta) && (
```

Comment:
Since this is duplicated, let's introduce a reusable constant `displayPayments` like it's done in [`FieldListDrawer.tsx`](https://github.com/opengovsg/FormSG/pull/6286/files#diff-2f4c1d0c5a445b55c5fdf1d5d4dc5bde69c281a75e4328f98478560ae7508f6c)

Reply:
Done in [c9be145](https://github.com/opengovsg/FormSG/pull/6286/commits/c9be145a81e1b1ef683e137debc7bdea73c66a5b)